plugin/method-channel.go

```diff
@@ -123,6 +123,11 @@ func (m *MethodChannel) HandleFuncSync(methodName string, f func(arguments inter
 	m.HandleSync(methodName, MethodHandlerFunc(f))
 }
 
+// HandleIgnore is used to ignore method call
+func (m *MethodChannel) HandleIgnore(arguments interface{}) (reply interface{}, err error) {
```

Comment:
I don't think HandleIgnore should be a method on the MethodChannel. It can just be a local function where-ever it's used..

Reply:
Where should the function be located?
At first I wanted it to be placed in /plugin.go but it didn't felt right.

Reply:
I would say just a local unexported function, in our case we use it in the textplugin.go so it can just be placed there I guess?

Reply:
Other channel like 'flutter/platform' can use `handleIgnore`.
 I'm not a big fan of hiding it in textplugin.go if it's used by other internal channels.

Reply:
Then its duplicated over there as well. I mean, we shouldn't make it public anyway.
Another way to implement the hiding could be to just: 
```go
p.channel.HandleFuncSync("TextInput.show", func(_ interface{})(interface{}, error){}) // ignored because <reason>
```

Reply:
I like the `func(_ interface{})(interface{}, error){}` approach!

Reply:
It will be more like `func(_ interface{}) (interface{}, error) { return nil, nil }`